.github/workflows/test_deploy.yml

```diff
@@ -840,6 +840,19 @@ jobs:
           PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
         run: make publish-lock-package
 
+  update-develop-branch:
+    runs-on: ubuntu-latest
+    if: "startsWith(github.ref, 'refs/tags/')"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: 'develop'
+      - name: merge changes from master
+        run: |
+          git merge master -X theirs
```

Comment:
I think we don't want `-X theirs`, better to fail on conflict than potentially lose work.

And when I tried locally, if there's no new commits on develop, there won't be conflicts.

Reply:
I'm pretty sure I had conflicts on the (few) changes we introduce when merging to master, but we can try without and see how it goes

Reply:
I tried it in two rounds. I did a 1.0.6 release like you, which unborked 1.0.5 and had conflicts. but then I added a dummy commit to develop, made a 1.0.7 release and got no conflicts.